docker_xylem/service.py

```diff
@@ -213,14 +272,18 @@ def _route_request(self, request):
         method = self.requestRouter.get(request.path)
 
         if not method:
+            self.log.warn(
+                '{request.path} is not implemented by plugin.',
+                request=request
+            )
             return "Not Implemented"
```

Comment:
We probably want to return HTTP 404 or something here. Does the docker volume driver documentation say anything about what docker expects?

(Note: This isn't about the logging. If we do decide to change the behaviour, let's open a separate ticket for that.)